README.md

```diff
@@ -10,6 +10,15 @@ This package implements approximation schemes for permutation and isometry invar
 Heavy use is made of trigonometric polynomials and spherical harmonics to obtain rotation invariance. There are also implementations of pure permutation invariant bases and of bases with only cylindrical symmetries for bond energies.
 Documentation is a work in progress; if you wish to use the code please contact the author.
 
+## Quickstart
+
+This package can be installed by first adding the ACE registry from the Julia package manager and pulling the ACE and JuLIP packages
+
+`] registry add https://github.com/ACEsuit`
```

Comment:
I think this is incorrect? We don't have an ACE registry. But we could add one. For the moment it is the MolSim registry that is needed.

Reply:
mind fixing that quickly then I'll merge.